compiler/ballerina-lang/src/main/resources/grammar/BallerinaParser.g4

```diff
@@ -522,7 +522,12 @@ returnStatement
     ;
 
 workerSendAsyncStatement
-    :   expression RARROW Identifier (COMMA expression)? SEMICOLON
+    :   expression RARROW workerIdentifier (COMMA expression)? SEMICOLON
+    ;
+
+workerIdentifier
```

Comment:
Spec use, following grammar rules. Shall we update like that?

peer-worker := worker-name | default
worker-name := identifier

Reply:
Will do the change in https://github.com/ballerina-platform/ballerina-lang/pull/14655. Thanks :)